app/src/standard/java/com/x8bit/bitwarden/data/platform/manager/CrashLogsManagerImpl.kt

```diff
@@ -20,6 +21,11 @@ class CrashLogsManagerImpl(
         set(value) {
             settingsRepository.isCrashLoggingEnabled = value
             Firebase.crashlytics.isCrashlyticsCollectionEnabled = value
+            if (value) {
```

Comment:
This class is injected into the `BitwardenApplication` and this will get invoked in the init block, so we should get setup immediately.

Reply:
I completely overlooked `trackNonFatalException`, below. We may not need Timber if we can inject `CrashLogsManager` in places we catch exceptions before showing the generic error dialog. Ultimately, we want non-fatal exceptions reported to Crashlytics when crash logging is enabled.